pages/linux/vboxmanage.md

```diff
@@ -0,0 +1,29 @@
+# VBoxManage
+
+> Command-line interface to VirtualBox.
+> Completely control VirtualBox from CLI, including all actions that the GUI provides and more.
```

Comment:
Not sure about this. Maybe we could reword it?

```suggestion
> Includes all the functionality of the GUI and more.
```

Reply:
Sure, less verbose and still clear.

Reply:
Looks good :+1: